### ISSUE TITLE
Bazel CI: rules_cc and rules_haskell is broken with Bazel@HEAD

### ISSUE THREAD

**Issue by meteorcloudy** [bazelbuild/bazel#10926]
https://buildkite.com/bazel/bazel-at-head-plus-downstream/builds/1417
```
Internal error thrown during build. Printing stack trace: java.lang.NullPointerException: //tests/haskell_proto_library:address_haskell_proto BuildConfigurationValue.Key[f0260d80912577e1bd92d9febed6b93f24862db1f34ffd977af1622a9f959e81] false
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:895)
	at com.google.devtools.build.skyframe.DelegatingWalkableGraph.getDirectDeps(DelegatingWalkableGraph.java:121)
	at com.google.devtools.build.lib.skyframe.SkyframeBuildView.assertSaneAnalysisError(SkyframeBuildView.java:773)
	at com.google.devtools.build.lib.skyframe.SkyframeBuildView.processErrors(SkyframeBuildView.java:616)
	at com.google.devtools.build.lib.skyframe.SkyframeBuildView.configureTargets(SkyframeBuildView.java:454)
	at com.google.devtools.build.lib.analysis.BuildView.update(BuildView.java:404)
	at com.google.devtools.build.lib.buildtool.AnalysisPhaseRunner.runAnalysisPhase(AnalysisPhaseRunner.java:213)
	at com.google.devtools.build.lib.buildtool.AnalysisPhaseRunner.execute(AnalysisPhaseRunner.java:124)
	at com.google.devtools.build.lib.buildtool.BuildTool.buildTargets(BuildTool.java:145)
	at com.google.devtools.build.lib.buildtool.BuildTool.processRequest(BuildTool.java:290)
	at com.google.devtools.build.lib.runtime.commands.BuildCommand.exec(BuildCommand.java:98)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.execExclusively(BlazeCommandDispatcher.java:564)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.exec(BlazeCommandDispatcher.java:208)
	at com.google.devtools.build.lib.server.GrpcServerImpl.executeCommand(GrpcServerImpl.java:603)
	at com.google.devtools.build.lib.server.GrpcServerImpl.lambda$run$2(GrpcServerImpl.java:659)
	at io.grpc.Context$1.run(Context.java:595)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
```

Culprit: https://github.com/bazelbuild/bazel/commit/906f5f1c233ca18c67a9cedd2c7182f33b88c44b
Identified by: https://buildkite.com/bazel/bazel-auto-sheriff-face-with-cowboy-hat/builds/83#790ec1a5-c10a-4486-9bc7-e968733b59db

**Comment by meteorcloudy** [comment]
@oquenchil Is there a quick fix? Otherwise, I'll do a rollback first.

**Comment by meteorcloudy** [comment]
Fixed d2701c88e8168f2843a7850896953489077d21db
